### DISCUSSION TITLE
Osu!Lazer stopped working

### DISCUSSION THREAD

**Discussion by Tcrafta** [ppy/osu#17748] [Q&A] [answered]
I just installed Osu!Lazer, when i started nothing happens and a windows error just pops up and say "Osu! has stopped working"

[runtime.log](https://github.com/ppy/osu/files/8457844/runtime.log).

**Comment by integerrr** [comment, marked as answer]
Lazer currently require Windows 8.1 at a minimum to run
https://github.com/ppy/osu#running-osu
